client/fc/fc.go

```diff
@@ -0,0 +1,35 @@
+package foundationcentral
+
+import (
+	"github.com/terraform-providers/terraform-provider-nutanix/client"
+)
+
+const (
+	libraryVersion = "v1"
+	absolutePath   = "api/fc/" + libraryVersion
+	userAgent      = "nutanix/" + libraryVersion
+)
+
+// Client manages the foundation central API
+type Client struct {
+	client *client.Client
+	Service
```

Comment:
Service should have a type right ? like `client *client.Client`

Reply:
No, it will like this. below service is pointing to *client.Client.